packages/build-ui/src/pages/collection/filter_sidebar.rs

```diff
@@ -0,0 +1,109 @@
+use bdk::prelude::*;
+
+use crate::pages::collection::FilterDropdown;
+
+
+
+#[component]
+pub fn FilterSidebar() -> Element {
+    rsx! {
+        div { class: "w-64 bg-[#000000] border-r border-[#333] p-4 flex flex-col gap-6",
+            // Artist Section
+            div { class: "flex flex-col gap-2",
```

Comment:
```suggestion
            div { id:"artist-section", class: "flex flex-col gap-2",
```

use `id` rather than comment.
it may help testing.